daemons/fenced/fenced_remote.c

```diff
@@ -1519,14 +1519,13 @@ report_timeout_period(remote_fencing_op_t * op, int op_timeout)
  * \internal
  * \brief Advance an operation to the next device in its topology
  *
- * \param[in,out] op      Operation to advance
- * \param[in]     device  ID of device just completed
- * \param[in]     msg     XML reply that contained device result (if available)
- * \param[in]     rc      Return code of device's execution
+ * \param[in] op      Fencer operation to advance
```

Comment:
out for op was true in a way that the structure is altered

Reply:
Yeah, I haven't been consistent about that because I've never been sure what doxygen intended for that case (the documentation isn't clear on that). I do agree that out probably makes more sense, but not going to worry about it here. Probably worth a small project one day to go through and make them all consistent.